docs/python 2 sunset.txt

```diff
@@ -0,0 +1,35 @@
+:orphan:
+
+Python 2 Sunset
+===============
+
+Since January 2020 and the release of Setuptools 45, Python 2 is no longer supported by the most current release (`discussion <https://github.com/pypa/setuptools/issues/1458>`_). Setuptools as a project continues to support Python 2 with bugfixes and important features on Setuptools 44.x.
```

Comment:
I've just checked some other doc files and it seems they wrap lines.  BTW, may I ask why setuptools use `txt` extension for docs for reST?

Reply:
Happy for someone to wrap the lines. I don't have a tool that does it for me automatically and I don't wish to invest the time to do it myself. Happy to accept a follow-on PR if someone wishes to expend the effort.

Reply:
I can take the job, it's literally 3 key stokes in my editor :smile:

Reply:
TIL, SublimeText does that for me too. I'll do that.